### ISSUE TITLE
a11y (critical) - Notification toggles are all missing labels

### ISSUE THREAD

**Issue by bmtcril** [openedx/frontend-app-account#1464]
For WCAG all form elemense must have labels, but the checkbox-toggles  for the "web" and "email" notification preferences have none, giving us ~14 violations on that page. More details here: https://github.com/openedx/end-to-end-tests/issues/49
